src/main/java/com/gachigang/ontherun/common/enums/CategoryName.java

```diff
@@ -0,0 +1,13 @@
+package com.gachigang.ontherun.common.enums;
+
+public enum CategoryName {
+    SALATES,
+    BURGERS,
+    SOUPS,
+    MEATS,
+    DRINKS;
+
```

Comment:
Why do we need this method if we have name() in Emun and it is public, I believe we can simply use it

Reply:
I try to find better solution

Reply:
Just use method name() from enum instead of your custom one

Reply:
I see